src/lib/exposes.ts

```diff
@@ -428,8 +428,8 @@ export class Light extends Base {
                 new Numeric('current_level_startup', access.ALL)
                     .withValueMin(1)
                     .withValueMax(254)
-                    .withPreset('minimum', 0, 'Use minimum permitted value')
-                    .withPreset('previous', 255, 'Use previous value')
+                    .withPreset('minimum', 'minimum', 'Use minimum permitted value')
+                    .withPreset('previous', 'previous', 'Use previous value')
```

Comment:
When using numeric preset values the presets wouldn't get the correct active state in the UI since the feature state is the string value (255 !== 'previous').

This shouldn't have any bearing on the actual state since the relevant fromZigbee/toZigbee always convert to the string value anyway.